spec/services/prison_visits/api_spec.rb

```diff
@@ -7,6 +7,7 @@
   # the VCR cassettes. Sadly this will vary from system to system as a result
   # of using UUIDs for prisons.
   let(:cardiff_prison_id) { '0614760e-a773-49c0-a29c-35e743e72555' }
+  let(:leeds_prison_id) { '8076d43d-429d-4e19-aac1-178ff9d112d3' }
```

Comment:
It shouldn't: 

https://github.com/ministryofjustice/prison-visits-2/blob/master/db/seeds/prison_uuid_mappings.yml